.github/workflows/warning_test.yaml

```diff
@@ -44,6 +44,7 @@ jobs:
       - name: Install
         run: |
           CFLAGS="-DCYTHON_TRACE=1 -DCYTHON_TRACE_NOGIL=1" pip install .
+          python setup.py build_ext --inplace
```

Comment:
this is interesting. We haven't needed this anywhere else. but obviously without it you had an issues. 
I'm willing to bet the real issue is the namespace degeneracy between the `pyuvdata` package and the directory with the same name. So in the next step pytest attempts to run tests from the directory.

I personally would prefer to remove this line, but add something in the next step that makes an empty directory and `cd`s into it.

Reply:
We can do it that way, that's the way it's done in other checks. The advantage of this is that I think it may cover more lines in the __init__.py where it figures out the version number. We currently have those lines marked with a `pragma: no cover` so it's not showing as more coverage, but I think they get hit with this approach. I can try removing the pragma to see.

Reply:
This line is what I always have to run on my own machine to be able to run tests from top level pyuvdata folder.